requirements.txt

```diff
@@ -1,3 +1,4 @@
 aiodns<4
 aiohttp<4
 aiohttp_jinja2<2
+tox==4.12.1
```

Comment:
do we really need precisely this version?
I suggest to have tox>=4.4.0 unless we need more recent features

Reply:
I don't think the version has to be exactly that. Will fix